WowUp.WPF/Utilities/StartupOptions.cs

```diff
@@ -0,0 +1,22 @@
+﻿using CommandLine;
+using System;
+using System.Collections.Generic;
+using System.Text;
+using WowUp.Common.Enums;
+
+namespace WowUp.WPF.Utilities
+{
+    public class StartupOptions
+    {
+        [Option(shortName: 'i', longName: "install", HelpText = "Specify addon urls to install them")]
+        public IEnumerable<string> InputURLs { get; set; }
+        [Option(shortName: 's', longName: "silent", HelpText = "Start application minimized")]
+        public bool Silent { get; set; }
+        [Option(shortName: 'u', longName: "update", HelpText = "Update non-ignored addons on startup")]
+        public bool Update { get; set; }
+        [Option(shortName: 'f', longName: "force", HelpText = "Update all addons on startup")]
+        public bool Force { get; set; }
+        [Option(shortName: 'c', longName: "client", HelpText = "Specify client version to use", Default = WowClientType.None)]
+        public WowClientType ClientType { get; set; }
+    }
```

Comment:
Can we add add another option with no shortName but a longName like "auto-update-shutdown" that could just run an auto update and close the app?

Reply:
Sure

Reply:
Not so sure there as well :)

Why introduce a single option for a specific scenario? Auto-updates will run on their own. I can add -s --shutdown to exit the app after all work is done. This can be used with/without the --silent and --update options.

Reply:
thats fair something like --minimized --exit/quit is fine

Reply:
Ok